templates/sharedfilesystem/resources/vkcs_sharedfilesystem_share_access.md.tmpl

```diff
@@ -14,9 +14,6 @@ description: |-
 ### NFS
```

Comment:
lets remove this
no other examples - no titles

Reply:
removed